index.js

```diff
@@ -67,7 +91,18 @@ function getAverageIMDBRating() {}
  *  countByRating(movies);
  *  //> { G: 3, PG: 7 }
  */
-function countByRating() {}
+function countByRating(movies) {
+  let movieRating = {};
```

Comment:
I would change this to `moviesRatings` since it contains the rating of many movies.